torchgeometry/imgwarp.py

```diff
@@ -14,11 +15,17 @@
 
 
 def normal_transform_pixel(height, width):
-    return torch.tensor([[
-        [2. / (width - 1), 0., -1.],
-        [0., 2. / (height - 1), -1.],
-        [0., 0., 1.]]
-    ])  # 1x3x3
+
```

Comment:
JIT returns the `height` and `width` as a one element tensor (instead of integer) and it cannot replace these values inside another tensor. It simply returns

```
[[0, 0, -1]
[0, 0, -1]
[0, 0, 1]]
```
as output. Hence, I constructed the tensor using arithmetic operations instead of replacing them into another tensor.

Reply:
I see. I think there is a clean alternative so that changes are minimal. If this two guys** are converted to `torch.tensor` and we assume that `normal_transform_pixel` consumes just tensor (need a type check there) I guess then that we solve the issue.

**https://github.com/arraiy/torchgeometry/blob/master/torchgeometry/imgwarp.py#L26-L27